src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -716,25 +702,44 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     // Creating the initial ACL must occur after the PASE session has adopted the fabric index
     // (see above) so that the concomitant event, which is fabric scoped, is properly handled.
     err = CreateAccessControlEntryForNewFabricAdministrator(commandObj->GetSubjectDescriptor(), fabricIndex,
```

Comment:
As a separate PR, we should change the last arg of `CreateAccessControlEntryForNewFabricAdministrator` to not be a NodeId, since that's not the type of caseAdminSubject.